src/components/common/BusinessContactInfo.vue

```diff
@@ -187,4 +188,19 @@ export default class BusinessContactInfo extends Mixins(CommonMixin) {
   font-weight: normal;
   color: $gray7;
 }
+
+// display the dotted line until extension is entered
+::v-deep .phone-ext .v-text-field > .v-input__control > .v-input__slot::before {
+  border-image: repeating-linear-gradient(
+    to right,
+    rgba(0, 0, 0, 0.38) 0px,
+    rgba(0, 0, 0, 0.38) 2px,
+    transparent 2px,
+    transparent 4px
+  ) 1 repeat;
+}
+
+::v-deep .phone-ext.phone-ext-filled .v-text-field > .v-input__control > .v-input__slot::before {
+  border-image: none;
+}
```

Comment:
This part is for ticket [13371](https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/bcgov/entity/13371). Two classes are added to tell the two different states of phone extension field: empty and filled.

Reply:
This part has been merged as of PR [13371 Create UI: Phone extension visual should remain "optional" until value entered](https://github.com/bcgov/business-create-ui/pull/436).